assets/large_language_models/rag/components/crack_and_chunk/spec.yaml

```diff
@@ -80,7 +80,7 @@ command: >-
   --input_data ${{inputs.input_data}}
   $[[--input_glob '${{inputs.input_glob}}']]
   $[[--allowed_extensions ${{inputs.allowed_extensions}}]]
-  --output_title_chunk ${{outputs.output_chunks}}
+  --output_chunks ${{outputs.output_chunks}}

```

Comment:
It will also need changes to pipelines?

Reply:
Looks like there is no reference
<img width="446" alt="image" src="https://github.com/Azure/azureml-assets/assets/99283778/8504af6b-b350-43e5-8244-8225b7afc4a0">